app/src/main/java/org/apache/fineract/ui/online/customers/customerlist/CustomersPresenter.java

```diff
@@ -27,15 +29,15 @@
 public class CustomersPresenter extends BasePresenter<CustomersContract.View>
         implements CustomersContract.Presenter {
 
-    private final DataManagerCustomer dataManagerCustomer;
+    private final ManagerCustomer dataManagerCustomer;
     private CompositeDisposable compositeDisposable;
 
     private int customerListSize = 50;
     private boolean loadmore = false;
 
     @Inject
     public CustomersPresenter(@ApplicationContext Context context,
-            DataManagerCustomer dataManager) {
+            DbManagerCustomer dataManager) {
```

Comment:
why DbManager?

Reply:
We are fetching data from the db not the api.